pytest/lib/proxy_instances.py

```diff
@@ -0,0 +1,30 @@
+import logging, multiprocessing
+from proxy import ProxyHandler, NodesProxy
+
+
+class RejectListHandler(ProxyHandler):
+
+    def __init__(self, reject_list, ordinal):
+        super().__init__(ordinal)
+        self.reject_list = reject_list
+
+    async def handle(self, msg, fr, to):
+        if fr in self.reject_list or to in self.reject_list:
+            msg_type = msg.enum if msg.enum != 'Routed' else msg.Routed.body.enum
+            logging.info(
+                f'NODE {self.ordinal} blocking message {msg_type} from {fr} to {to}')
```

Comment:
Compared to Michael's original PR I changed logging here to include the message type.